torch/nn/quantized/modules/conv.py

```diff
@@ -1,4 +1,6 @@
 # coding=utf-8
+r"""Quantized convolution modules."""
```

Comment:
why do we need this comment?

Reply:
These are "package-level docstrings". Ideally we want to have them in all importable files. That way, the user can run something like

```python
from torch.nn.quantized.modules import conv
help(conv)
```

And that would present the docstring. Ideally we want to have more info in there. If this is not the pattern we are following, I can remove it.